king_phisher/client/graphs.py

```diff
@@ -275,12 +272,38 @@ def signal_activate_popup_menu_export(self, action):
 		destination_file = response['target_path']
 		self.figure.savefig(destination_file, format='png')
 
+	def signal_activate_popup_refresh(self, event):
+		self.refresh()
+
 	def signal_toggled_popup_menu_show_toolbar(self, widget):
 		if widget.get_property('active'):
 			self.navigation_toolbar.show()
 		else:
 			self.navigation_toolbar.hide()
 
+	def resize(self, width=0, height=0):
+		"""
+		Attempt to resize the canvas. Regardless of the parameters the canvas
+		will never be resized to be smaller than :py:attr:`.minimum_size`.
+		:param int width: The desired width of the canvas.
```

Comment:
All of the doc strings should have a single blank line between their descriptions and parameters.